docs/application/tizen-studio/setup/install-sdk.md

```diff
@@ -100,6 +167,394 @@ package-manager-cli install [--accept-license] [--no-java-check] [--proxy <value
+| `-act`                      | Activate extra server. |
+| `-dact`                     | Deactivate extra server.|
+| `index[,...]`               | Index of extra servers. |
+
+**Examples:**
+
+**To get the index of different servers**
+
+Run the command using the following syntax:
+
+- Windows&reg;, Ubuntu and macOS:
+
+  ```
+  > package-manager-cli extra --list
+
+  Package Manager (0.5.39)
+   Preparing...
+   Validating...
+   Index        : 1
+   Name         : Samsung Certificate Extension
+   Repository   : https://download.tizen.org/sdk/extensions/tizen-certificate-extension_2.0.61.zip
+   Default      : true
+   Activate     : true
+
+   Index        : 2
+   Name         : Samsung Wearable Extension
+   Repository   : https://developer.samsung.com/sdk-manager/repository/tizen-wearable-extension-sdk-1.5.0.zip
+   Default      : true
+   Activate     : true
+
+   Index        : 3
+   Name         : Samsung Tizen TV SDK
+   Repository   : https://sdf.samsungcloudcdn.com/Public/smart_tv_sdk/releases/samsung_tizen_studio_tv_sdk/stv_ext_public/TV
+   Default      : true
+   Activate     : true
+
+   Index        : 4
+   Name         : Tizen IoT Headed
+   Repository   : https://download.tizen.org/sdk/extensions/Tizen_IoT_Headed
+   Default      : true
+   Activate     : true
+
+   Index        : 5
+   Name         : Tizen IoT Headless
+   Repository   : https://download.tizen.org/sdk/extensions/Tizen_IoT_Headless
+   Default      : true
+   Activate     : true
+
+   Index        : 6
+   Name         : Tizen IoT Setup Manager
+   Repository   : https://download.tizen.org/sdk/extensions/iot-setup-mgr
+   Default      : false
+   Activate     : true
+
+  ```
+Run the act/dact command using the above index:
+- Example: extra -act command
+   ```
+   > package-manager-cli extra -act 1
+
+   Package Manager (0.5.39)
+   Preparing...
+   Validating...
+
+   Extension         : Samsung Certificate Extension
+   Activation status : true
+  ```
+- Example: extra -dact command
+   ```
+   > package-manager-cli extra -dact 1
+
+   Package Manager (0.5.39)
+   Preparing...
+   Validating...
+
+   Extension         : Samsung Certificate Extension
+   Activation status : false
+   ```
+
+## Remove existing extra server with CLI
+
+Run the CLI Package Manager with the `extra` command using the following syntax:
+
+```
+package-manager-cli extra --remove <index[,...]>
+```
+**Table: Activate/Deactivate command parameters**
+
+| Parameter                   | Description                              |
+|-----------------------------|------------------------------------------|
+| `--remove`                  | Remove existing extra server |
+| `index[,...]`               | Index of extra servers. |
+
+**Examples:**
+
+**To get the index of different servers, use the following command:**
+
+- Windows&reg;, Ubuntu, and macOS:
+
+  ```
+  > package-manager-cli extra --list
+  ```
+
+**To remove existing extra server**
+
```

Comment:
```suggestion

Run the commands using the following syntax:

```

Reply:
I have made the changes here exactly as you suggested, but this suggested change is still showing.

Reply:
@abhishek-kp1 , don't worry about it, the change has been properly implemented and I've resolved it.